cg/meta/workflow/raredisease.py

```diff
@@ -142,3 +151,46 @@ def get_workflow_metrics(self, sample_id: str) -> dict:
     @staticmethod
     def set_order_sex_for_sample(sample: Sample, metric_conditions: dict) -> None:
         metric_conditions["predicted_sex_sex_check"]["threshold"] = sample.sex
+
+    def get_sample_coverage_file_path(self, bundle_name: str, sample_id: str) -> str | None:
+        """Return the Raredisease d4 coverage file path."""
+        coverage_file: File | None = self.housekeeper_api.get_file_from_latest_version(
+            bundle_name=bundle_name, tags=RAREDISEASE_COVERAGE_FILE_TAGS + [sample_id]
+        )
+        if coverage_file:
+            return coverage_file.full_path
+        LOG.warning(f"No coverage file found with the tags: {RAREDISEASE_COVERAGE_FILE_TAGS}")
+        return None
+
+    def get_gene_ids_from_scout(self, panels: list[str]) -> list[int]:
+        """Return HGNC IDs of genes from specified panels using the Scout API."""
+        gene_ids = []
+        for panel in panels:
+            genes: list[dict] = self.scout_api.get_genes(panel)
+            gene_ids.extend(
+                gene.get("hgnc_id") for gene in genes if gene.get("hgnc_id") is not None
```

Comment:
```suggestion
                gene.get("hgnc_id") for gene in genes if gene.get("hgnc_id") is not None
```
"hgnc_id" create constant